src/utils/addresses.ts

```diff
@@ -31,33 +34,27 @@ export const checkAddressValidity = (address: string) => {
   return match[0] === address && address
 }
 
-export const loadStoredAddressesInfoOfAccount = (username: string): AddressInfo[] => {
-  const storedAddressIndexes = localStorage.getItem(`${username}-address-info`)
+export const loadStoredAddressesIndexAndSettingsOfAccount = (username: string): AddressIndexAndSettings[] => {
```

Comment:
That's a big fat name, dammn 🤩
We may add other keys in the future, so I'm thinking that the types `StoredAddressInfo` or `StoredAddressMetadata` would be more convenient? And thus `loadStoredAddressesInfo` or `loadStoredAddressesMetadata`?

Reply:
Ah, yes, good old naming problem! Thanks fo the `StoredAddressMetadata`, I really like it! I didn't like my initial `AddressInfo` idea, because there is a `AddressInfo` type in `alephium-js`. Metadata sounds great.